### PR TITLE
Add live code samples to <a> element page

### PR DESCRIPTION
**Pull request by Mehrm0**
### Description
Fixed some examples(Live Samples)

### CODE REVIEW

**Review by queengooborg — CHANGES_REQUESTED** [review]
It looks like your changes contain additional (and unwanted) changes, such as reverting link changes and adding back tags.  Could you revert those changes, please?

**Review by Mehrm0 — COMMENTED** [review]
Unwanted **Guides** in links and tags are removed. But I don't understand why were they added, I didn't add them.

File: files/en-us/web/html/element/a/index.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
...
+  - Element
+  - HTML
+  - HTML text-level semantics
+  - HTML:Flow content
+  - HTML:Interactive content
+  - HTML:Palpable Content
+  - HTML:Phrasing content
+  - Inline element
+  - Reference
+  - Web
```
Comment (queengooborg) [inline files/en-us/web/html/element/a/index.md:16]: Tags should be removed as well!

```suggestion
```

**Review by queengooborg — DISMISSED** [review]

**Comment by queengooborg** [comment]
Thank you, I'll pass it off to another for the final review!

File: files/en-us/web/html/element/a/index.md [hunk 2]
```diff
@@ -244,7 +256,7 @@ document
   );
 ```
 
-##### Result
+#### Result
```
Comment (hamishwillee) [inline files/en-us/web/html/element/a/index.md:259]: Nope, this is now too shallow - not as deep as its parent.

Comment (hamishwillee) [reply files/en-us/web/html/element/a/index.md:259]:
```suggestion
##### Result
```

File: files/en-us/web/html/element/a/index.md [hunk 3]
```diff
@@ -268,6 +280,9 @@ A sadly common mistake is to only link the words "click here" or "here":
 <p>Learn more about our products <a href="/products">here</a>.</p>
 ```
 
+#### Result
+{{EmbedLiveSample('Inaccessible, weak link text')}}
```
Comment (hamishwillee) [inline files/en-us/web/html/element/a/index.md:284]: This should be at deeper level than its parent.
Also spaces between heading and content.

```suggestion
##### Result

{{EmbedLiveSample('Inaccessible, weak link text')}}
```

File: files/en-us/web/html/element/a/index.md [hunk 4]
```diff
@@ -276,6 +291,9 @@ Luckily, this is an easy fix, and it's actually shorter than the inaccessible ve
 <p>Learn more <a href="/products">about our products</a>.</p>
 ```
 
+#### Result
+{{EmbedLiveSample('Strong link text')}}
```
Comment (hamishwillee) [inline files/en-us/web/html/element/a/index.md:295]: Spacing and nesting levels 
```suggestion
##### Result

{{EmbedLiveSample('Strong link text')}}
```

File: files/en-us/web/html/element/a/index.md [hunk 5]
```diff
@@ -300,6 +318,9 @@ People experiencing low vision conditions, navigating with the aid of screen rea
 </a>
 ```
 
+#### Result
+{{EmbedLiveSample('Link that opens a new tab/window')}}
```
Comment (hamishwillee) [inline files/en-us/web/html/element/a/index.md:322]:
```suggestion
##### Result

{{EmbedLiveSample('Link that opens a new tab/window')}}
```

File: files/en-us/web/html/element/a/index.md [hunk 6]
```diff
@@ -320,6 +341,9 @@ If an icon is used to signify link behavior, make sure it has {{HTMLAttrxRef("al
 </a>
 ```
 
+#### Result
+{{EmbedLiveSample('Link to a non-HTML resource')}}
```
Comment (hamishwillee) [inline files/en-us/web/html/element/a/index.md:345]:
```suggestion
##### Result

{{EmbedLiveSample('Link to a non-HTML resource')}}
```

File: files/en-us/web/html/element/a/index.md [hunk 7]
```diff
@@ -170,6 +178,10 @@ For details about `mailto:` URLs, such as including a subject or body, see [Emai
 <a href="tel:+1(800)555-0123">(800) 555-0123</a>
 ```
 
+#### Result
+{{EmbedLiveSample('Linking to telephone numbers')}}
```
Comment (hamishwillee) [inline files/en-us/web/html/element/a/index.md:182]: Space between heading and content. The nesting level is fine though - parent is 3, so this is 4.

```suggestion
#### Result

{{EmbedLiveSample('Linking to telephone numbers')}}
```

File: files/en-us/web/html/element/a/index.md [hunk 8]
```diff
@@ -160,6 +165,9 @@ To create links that open in the user's email program to let them send a new mes
 ```html
 <a href="mailto:nowhere@mozilla.org">Send email to nowhere</a>
 ```
+#### Result
```
Comment (hamishwillee) [inline files/en-us/web/html/element/a/index.md:168]: Space between heading and content.
```suggestion

#### Result
```

**Review by hamishwillee — APPROVED** [review]
Looks good. Thank you.

File: files/en-us/web/html/element/a/index.md [hunk 9]
```diff
@@ -151,6 +151,11 @@ a {
 <h2 id="Section_further_down">Section further down</h2>
 ```
 
+#### Result
+
+{{EmbedLiveSample('Linking to an element on the same page')}}
+
+
 > **Note:** You can use `href="#top"` or the empty fragment (`href="#"`) to link to the top of the current page, [as defined in the HTML specification](https://html.spec.whatwg.org/multipage/browsing-the-web.html#scroll-to-the-fragment-identifier).
```
Comment (hamishwillee) [inline files/en-us/web/html/element/a/index.md:159]: Make linter happy
```suggestion
{{EmbedLiveSample('Linking to an element on the same page')}}

> **Note:** You can use `href="#top"` or the empty fragment (`href="#"`) to link to the top of the current page, [as defined in the HTML specification](https://html.spec.whatwg.org/multipage/browsing-the-web.html#scroll-to-the-fragment-identifier).
```

File: files/en-us/web/html/element/a/index.md [hunk 10]
```diff
@@ -170,6 +179,11 @@ For details about `mailto:` URLs, such as including a subject or body, see [Emai
 <a href="tel:+1(800)555-0123">(800) 555-0123</a>
 ```
 
+#### Result
+
+{{EmbedLiveSample('Linking to telephone numbers')}}
+
+
 `tel:` link behavior varies with device capabilities:
```
Comment (hamishwillee) [inline files/en-us/web/html/element/a/index.md:187]: Make linter happy
```suggestion
{{EmbedLiveSample('Linking to telephone numbers')}}

`tel:` link behavior varies with device capabilities:
```

**Comment by hamishwillee** [comment]
Thanks @Mehrm0 !

**Pull request merged**